spec.md

```diff
@@ -276,6 +274,23 @@ help intermediate gateways determine how to route the events.
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### datacontentencoding
+* Type: `String` per [RFC 2045 Section 6.1](https://tools.ietf.org/html/rfc2045#section-6.1)
+* Description: Describes the content encoding for the `data` 
+  attribute for when the `data` field MUST be encoded as a string,
+  like with structured transport binding modes using the JSON event 
+  format, but the `datacontenttype` indicates a non-string media 
+  type. When the `data` field's effective data type is not `String`,
```

Comment:
> When the `data` field's effective data type is not `String`,
>  this attribute MUST NOT be set and MUST be ignored when set.

why not? I want to zip my http binary data payload...

Reply:
Or I want to base64 encode my binary http payload because I am a weirdo...

Reply:
Perhaps not, looking at RFC 2045 I do not see anything I would want to use besides base64.  And normal http transports can give me gzip... 

/lgtm